core/contracts/oracle/implementation/TokenMigrator.sol

```diff
@@ -48,7 +51,7 @@ contract TokenMigrator {
             return;
         }
 
-        FixedPoint.Unsigned memory newBalance = oldBalance.div(rate);
+        FixedPoint.Unsigned memory newBalance = oldBalance.divCeil(rate);
```

Comment:
Not necessarily against this change, but curious why you chose to ceil this?

One minor issue I see with ceiling this is that if rate is >> 1, rounding errors could allow token holders to split up their tokens (pre-snapshot) in such a way to generate "free" new tokens. In most cases 1 wei of new tokens wouldn't be worth the gas for an extra transaction, but theoretically there are cases where that amount might matter. On the other hand, if we round down, you can only lose tokens by splitting them up.

Curious to hear everyone's thoughts. @ptare @chrismaree

Reply:
@mrice32 interesting I hadn't thought of that. To be honest, this change was only to mitigate downside for tiny token holders. In practice, it wouldn't really be an issue and would more often result in normal token holders gaining an extra wei of tokens via rounding. Happy to revert this but curious what @ptare and @chrismaree think

Reply:
Thinking more about this, I think this `divCeil` creates more upside risk (token holders getting more tokens than they should post-migration) than downside risk, I'm going to revert back to `div` but still curious what @ptare and @chrismaree opine

Reply:
I don't have a strong opinion, probably won't matter much either way.

Reply:
ye, I agree that `div` makes the most sense here but also not super strong feelings.